# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/bsd-games/delay_output-sym.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' bsd-games-2.14/hack/extern.h bsd-games-2.14-fix/hack/extern.h
--- bsd-games-2.14/hack/extern.h	2003-03-28 11:51:54.000000000 +0100
+++ bsd-games-2.14-fix/hack/extern.h	2003-10-02 03:56:10.000000000 +0200
@@ -511,7 +511,7 @@
 void standoutend __P((void));
 void backsp __P((void));
 void bell __P((void));
-void delay_output __P((void));
+void delay_output_my __P((void));
 void cl_eos __P((void));
 
 /* hack.timeout.c */
diff -urN -x '*~' bsd-games-2.14/hack/hack.pri.c bsd-games-2.14-fix/hack/hack.pri.c
--- bsd-games-2.14/hack/hack.pri.c	2003-03-29 13:05:30.000000000 +0100
+++ bsd-games-2.14-fix/hack/hack.pri.c	2003-10-02 03:56:22.000000000 +0200
@@ -171,7 +171,7 @@
 		return;
 	}
 	if (prevx >= 0 && cansee(prevx, prevy)) {
-		delay_output();
+		delay_output_my();
 		prl(prevx, prevy);	/* in case there was a monster */
 		at(prevx, prevy, levl[prevx][prevy].scrsym);
 	}
diff -urN -x '*~' bsd-games-2.14/hack/hack.termcap.c bsd-games-2.14-fix/hack/hack.termcap.c
--- bsd-games-2.14/hack/hack.termcap.c	2003-03-29 13:05:30.000000000 +0100
+++ bsd-games-2.14-fix/hack/hack.termcap.c	2003-10-02 03:56:30.000000000 +0200
@@ -326,7 +326,7 @@
 }
 
 void
-delay_output()
+delay_output_my()
 {
 
 	/* delay 50 ms - could also use a 'nap'-system call */
